.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -26,51 +26,66 @@ body:
     attributes:
       label: Steps to reproduce
       placeholder: |
+        E.g., What happened, and what did you expect to happen? Add images,
+        GIFs, and videos if you have them on hand.
+
         1. Start the Studio app.
         2. Click on 'Add site'.
         3. ...
+
+        Add any information that may be relevant, such as:
+          - Browser/Platform
+          - Theme
+          - Logs/Errors
     validations:
       required: true
-  - type: textarea
-    id: expected
+
+  - type: markdown
     attributes:
-      label: What you expected to happen
-      placeholder: |
-        e.g. The new site form shows up.
-    validations:
-      required: true
-  - type: textarea
-    id: actual
+      value: |
+        <br>
+
+        ## Impact
+        Please help us understand more about the impact of this issue to help determine next steps.
+        If you are unsure about anything, please use your judgment to make an educated guess.
+  - type: dropdown
+    id: impact
     attributes:
-      label: What actually happened
-      placeholder: |
-        e.g. Clicking the 'Add site' button does nothing.
+      label: Site owner impact
+      description: Approximately what percentage of the total users of the platform are impacted? Unsure? Please provide your most educated guess!
+      options:
+        - Less than 20% of all
+        - Between 20% and 60% of all
+        - More than 60% of all
     validations:
       required: true
   - type: dropdown
-    id: users-affected
+    id: severity
     attributes:
-      label: Impact
-      description: Approximately how many users are impacted?
+      label: Severity
+      description: What is the severity of this issue? Please take a look at the descriptions below for further context.<br>
+        <br> - **Critical:** Prevents core functionality or has severe impact on the website/platform.
```

Comment:
Should we say 'app' instead of website/platform here and in other places?

Reply:
I think we can change all references to "the website/platform" to "the website, app, or platform" to help make the categorisation more accommodating.

Reply:
Actually, is it supposed to say that an "issue has a severe impact on one of website, app or platform" or "website or app issue has a severe impact when using the given platform"? Note that at the bottom, we define platform as one of the supported operating systems.